server/neptune/workflows/internal/terraform/state/workflow.go

```diff
@@ -0,0 +1,28 @@
+package state
+
+import "net/url"
+
+type JobStatus string
+
+const WorkflowStateChangeSignal = "workflow-state-change"
```

Comment:
nit: since the signal name is within the context of atlantis namespace and we'll see it in the temporal UI, should we make it more specific and call terraform-workflow-state-change?

Reply:
yeah i can change it.